resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -142,6 +142,16 @@ package object config {
       .stringConf
       .createOptional
 
+  private[spark] val KUBERNETES_DRIVER_POD_PORTS =
+    ConfigBuilder("spark.kubernetes.driver.ports")
```

Comment:
`spark.kubernetes.driver.additionalPorts` ?

Want to make it clear these are in addition to the default ports.  Or is there a pattern we should be following from YARN / Mesos?